src/penguinv/cpu_identification.h

```diff
@@ -17,6 +17,9 @@
             #if _MSC_VER >= 1700
                 #define PENGUINV_AVX_SET
             #endif
+            #if _MSC_VER >= 1911
```

Comment:
Hi @0x72D0 I modified the condition here as logically it should be this way.

Reply:
Yeah I agree it's better this way :)